docs/dev-env.md

```diff
@@ -73,6 +73,25 @@ To use a different shell for `nix develop`, try `nix develop -c zsh`. You can al
 don't want to remember about it. That's the recommended way to use a different shell
 for `nix develop`.
 
+## Cachix binary cache
+
+Fedimint uses a [Cachix](https://www.cachix.org/) binary cache to cache builds.
+To benefit from this cache and avoid building everything from scratch, you must
+ensure that your user is a [trusted user](https://nixos.org/manual/nix/stable/command-ref/conf-file.html#conf-trusted-users).
+You can do this by modifying `/etc/nix/nix.conf`, adding the following line.
+
+```
+trusted-users = the_name_of_your_user
+```
+
+Alternatively, if you do not want to add your user to the list of trusted users, you can
+run the following command, which will add <https://fedimint.cachix.org> and its public key
+to your nix configuration.
+
+```
+nix develop .#bootstrap -c cachix use fedimint
```

Comment:
I don't think this actually adds user to trusted-users... . I don't see it anywhere in the code.

Reply:
That's not what the comment claims, the command edits `~/.config/nix/nix.conf` and adds the binary substituter there.

Reply:
Does it thought?

Reply:
To clarify, the section is saying "there are two ways to use the binary cache: either add yourself to trusted-users, or run this command (which happens to modify nix.conf in a different way)"

> Does it thought?

Yeah so for me, before adding my user to `trusted-users`, I ran `cachix use fedimint` and my `/etc/nix/nix.conf` was updated. It added <https://fedimint.cachix.org> to `trusted-substituters` and added the public key to `trusted-public-keys`

Looks like the fallback is `/etc/nix/nix.conf` for untrusted users: https://docs.cachix.org/faq#what-happens-when-i-run-cachix-use-both-immediately-and-any-stateful-effects-for-the-future. 

Doesn't make much sense to me- I haven't had to use a non-nixpkgs binary cache before so this is all new to me.

Reply:
> Does it thought?

For me it did, I removed cachix from the file and ran `watch cat ~/.config/nix/nix.conf` and saw it happen ^^